website/common/locales/en/spells.json

```diff
@@ -36,7 +36,7 @@
   "spellRogueStealthText": "Stealth",
   "spellRogueStealthNotes": "With each cast, a few of your undone Dailies won't cause damage tonight. Their streaks and colors won't change. (Based on: PER)",
   "spellRogueStealthDaliesAvoided": "<%= originalText %> Number of dailies that will be avoided: <%= number %>.",
-  "spellRogueStealthMaxedOut": "You have already avoided all your dailies; there's no need to cast this again.",
+  "spellRogueStealthMaxedOut": "Using this skill won't have any additional effect.",
```

Comment:
Yeah, makes sense to me

Reply:
@Mike-Antonacci the way to do this is to remove the unused string from the `website/common/locales/en/spells.json` file and then to change all the instances where it's used across the site with the other one

Reply:
I deleted the unused string, and it looks like all instances are for other languages, which seems odd to me, I would've expected that this string would be used at least somewhere. (`git grep "spellRogueStealthMaxedOut"` only showed other languages, for example `website/common/locales/ar/spells.json:`). I also successfully did `git checkout origin/develop -- package-lock.json` as you said.

I thought I committed the changes, and I thought I pushed them, but it said `no changes added to commit (use "git add" and/or "git commit -a")` and I don't know what that means or how to resolve it. Thank you for your patience, this is my very first pull request.

Reply:
@Mike-Antonacci what does `git status` says?

Reply:
The idea is that you remove the line for `spellRogueStealthMaxedOut` from the english file, git status will tell you that a file was changed, then you can commit the change with `git commit -am'an informative message'` and push it

Reply:
`git status` says `no changes added to commit (use "git add" and/or "git commit -a")`

Reply:
Okay, now I successfully committed and pushed.

Reply:
It's possible that if git status says there are no changes that it got added in a different commit, otherwise it would see the difference between what's stored in git and the saved version of the file on your computer

Unfortunately it's a bit difficult to debug these problems from github comments so if you prefer I'm happy to apply the changes here

Reply:
Fully agreed that it's difficult to debug via comments. If you could apply the changes yourself, that would be great. Thank you for that, and thank you for all you do for Habitica

Reply:
the string has been removed, I'm not sure why it's still showing the diffs in the package-lock files because they're exactly the same as `develop`

Reply:
not sure how but it got fixed 🎉 The PR should be ready now, I'll merge it tomorrow